core/tests/protractor_utils/LibraryPage.js

```diff
@@ -100,7 +100,7 @@ var LibraryPage = function() {
       '.protractor-test-exp-summary-tile-title'))).perform();
 
     await waitFor.elementToBeClickable(
-      addToPlaylistButton, 'Add to playlist Icon taking too long to load');
+      addToPlaylistButton, 'Watch later Icon taking too long to load');
```

Comment:
Actually, rename the button too right? If we remove references to "add to playlist", lets do it throughout.

Reply:
Done! I have made the changes.